internal/pkg/cli/env.go

```diff
@@ -26,6 +26,7 @@ An environment represents a deployment stage.`,
 	cmd.AddCommand(BuildEnvInitCmd())
 	cmd.AddCommand(BuildEnvListCmd())
 	cmd.AddCommand(BuildEnvDeleteCmd())
+	cmd.AddCommand(BuildEnvShowCmd())
```

Comment:
We probably should remove this for now so we don't release an empty command next release :P

Reply:
Ah, got it. Thanks!

Reply:
Commented it out instead of deleting so I don't forget to reinstate it eventually. :)